src/main/scala/com/nekokittygames/Thaumic/Tinkerer/common/tiles/TileBoundJar.scala

```diff
@@ -30,6 +30,7 @@ class TileBoundJar  extends TileJarFillable{
         amount = aspectList.getAmount(aspect)
       }
       super.update()
+      this.worldObj.markBlockForUpdate(this.pos);
```

Comment:
I dont feel right about this, I will make a network changed event, all the jars can listen for that, and only update when they need to, saves some network spam